artiq/frontend/artiq_master.py

```diff
@@ -155,7 +155,10 @@ def ccb_issue(service, *args, **kwargs):
     atexit_register_coroutine(server_logging.stop)
 
     print("ARTIQ master is now ready.")
-    loop.run_forever()
+    try:
+        loop.run_forever()
+    except KeyboardInterrupt:
```

Comment:
As I said, the proper way to close the master is to send a termination request via ``artiq_client``. Support for it could be added to ``artiq_dashboard``.

Reply:
actually I can not find this feature in the output of `artiq_client --help`. am I just badly overlooking something?